src/a.tex

```diff
@@ -195,6 +195,23 @@ \section{Load-Reserved/Store-Conditional Instructions}
 A platform specification may constrain the size and shape of the reservation
 set.  For example, the Unix platform is expected to require the set be
 contiguous and no greater than the virtual memory page size.
+
+Reservation sets provide implementation leeway to manage reservations
+at a granularity matching hardware design.  A common implementation
+is to map reservation sets in cacheable space to cache lines,
+harmonizing reservation invalidation with the cache coherence protocol.
+
+It is software's joint responsibility to ensure forward progress, for
+example by avoiding the allocation of data subject to progress-impeding
+writes in the same reservation set(s) as the LR/SC word.
+
+Where this is not a concern, an implementation might define a
+single reservation set covering the entire address space.
+At the other extreme, a suitably constrained system might define
+byte-sized reservation sets, relieving software of the need for
```

Comment:
should be "word-sized" instead of "byte-sized", given that misaligned LR/SC is not supported and the minimum LR/SC size is one word.